MATLAB/+Battery/InputEqnv1.m

```diff
@@ -0,0 +1,28 @@
+function Output = InputEqn(parameters,t,inputParameters)
+
+persistent input inputparameters time;
+
+if isempty(input) || ~isequal(inputparameters,inputParameters)

```

Comment:
The function should handle the case where the inputParameters argument is not specified (see the original InputEqn).